apps/trading/e2e/README.md

```diff
@@ -0,0 +1,78 @@
+# Console-Test
+
+`console-test` is a repository containing end-to-end tests for a console application using vega-market-sim. This README will guide you through setting up your environment and running the tests.
+
+## Prerequisites
+
+- [Poetry](https://python-poetry.org/docs/#installing-with-the-official-installer)
+- [Docker](https://www.docker.com/)
+
+## Getting Started
+
+1. **Install Poetry**: Follow the instructions on the [official Poetry website](https://python-poetry.org/docs/#installing-with-the-official-installer).
+
+2. **Install Docker**: Follow the instructions on the [official Docker website](https://docs.docker.com/desktop/).
+
+3. **Set Up a Poetry Environment**:
+   Execute the command below to configure the Poetry environment.
+
+   ```bash
+   nom run trading:test:install
+   ```
+
+4. **Download Necessary Binaries**:
+   Use the following command within your Python environment. The `--force` flag ensures the binaries are overwritten, and the `--version` specifies the desired version.
+   ```bash
+   python -m vega_sim.tools.load_binaries --force --version $VEGA_VERSION
```

Comment:
This doesn't explain how to get the `vega_sim.tools.load_binaries` module.

Reply:
It looks like it lacks of `requirements.txt`, doesn't it?

Reply:
We shouldn't need a requirements.txt. The poetry lock sort of does the same.

I think the npm command for doing poetry install doesn't seem to always play nice, which is why this python command may not work.

I could add the actual commands required to do it all, rather than using the npm commands